statsmodels/stats/stattools.py

```diff
@@ -76,27 +76,41 @@ def omni_normtest(resids, axis=0):
 
 
 def jarque_bera(resids, axis=0):
-    """
-    Calculate residual skewness, kurtosis, and do the JB test for normality
+    r"""
+    Calculates the Jarque-Bera test for normality
 
     Parameters
     -----------
-    resids : array-like
+    data : array-like
+        Data to test for normality
     axis : int, optional
-        Default is 0
+        Axis to use if data has more than 1 dimension. Default is 0
 
     Returns
     -------
-    JB, JBpv, skew, kurtosis
+    out : tuple (JB, JBpv, skew, kurtosis)
```

Comment:
python always returns a tuple when there are several return values. So this should be standard format with `JB, JBpv, skew, kurtosis` separately documented as returns